README.md

```diff
@@ -12,6 +12,107 @@ OpenSearch Ruby Client
 
 **opensearch-ruby** is [a community-driven, open source fork](https://aws.amazon.com/blogs/opensource/introducing-opensearch/) of elasticsearch-ruby licensed under the [Apache v2.0 License](LICENSE.txt).
 For more information, see [opensearch.org](https://opensearch.org/).
+
+## Setup
+
+To add the client to your project, install it using [RubyGem](https://rubygems.org/)
+
+`gem install opensearch`
+
+Import the client as a module:
+
+`require 'opensearch'`
+
+If you prefer to add the client manually or just want to examine the source code, see [opensearch-ruby on GitHub](https://github.com/opensearch-project/opensearch-ruby).
+
+## Sample code
+
+```
+require 'opensearch'
+
+client = Opensearch::Client.new url: 'http://localhost:9200', log: true
+
+# If you want to use authentication credentials
+# client = Opensearch::Client.new url: 'http://USERNAME:PASSWORD@myhost:8080', log: true
```

Comment:
Shall we use default opensearch distribution ( with security ) for example
```suggestion
client = Opensearch::Client.new url: 'https://admin:admin@localhost:9200', log: true
```

Reply:
Fixed